_posts/2019-06-07-python-datascience-with-openfaas.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Serving Data Science Models with OpenFaaS
+# TODO
+description: "Build and deploy a name classification function with OpenFaaS [<--FUN SUGGESTIONS WELCOME]"
+date: 2019-06-01
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
```

Comment:
Is this the same photo as the Python post? Can you show something with connections / neural nets?

Reply:
I just have not found a picture yet